Controllers/AdminController.php

```diff
@@ -135,6 +135,27 @@ public function postEdit(int $postId = null): void
             );
     }
 
+    /**
+     * Controller method to display the list of the posts in admin session
+     *
+     * @return void
+     *
+     * @throws \Twig_Error_Loader
+     * @throws \Twig_Error_Runtime
+     * @throws \Twig_Error_Syntax
+     */
+    public function postList(): void
+    {
+         $postList = Post::getPostList();
```

Comment:
Au temps pour moi, mais il ne manquerait pas comme un `$this->redirectToLoginIfNotConnected()` au début de chacune de tes méthodes de l'admin Controller ? :-)